resources/sass/app.scss

```diff
@@ -242,6 +250,14 @@ legend {
     }
 }
 
+.list-bios {
+    @extend %index-list;
```

Comment:
I would say should just use the same component class for both of these if one is extending the other, and add another class for the `h3`s.

Makes me think of the ever brilliant Nicholas Gallagher article: http://nicolasgallagher.com/about-html-semantics-front-end-architecture/

> The most reusable components are those with class names that are independent of the content.